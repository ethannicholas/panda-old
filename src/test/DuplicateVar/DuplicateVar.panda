var name := "Hello"
for i in 0 .. 11 {
    for i in 0 .. 6 {
        Console.writeLine(i)
    }
}
for i in 0 .. 10 {
    var name := "Goodbye"
    Console.writeLine(name)
}
var name := "Done!"
