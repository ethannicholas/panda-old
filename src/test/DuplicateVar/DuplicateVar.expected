error: DuplicateVar.panda:3:9: 'i' was already defined at DuplicateVar.panda:2:5
error: DuplicateVar.panda:8:9: 'name' was already defined at DuplicateVar.panda:1:5
error: DuplicateVar.panda:11:5: 'name' was already defined at DuplicateVar.panda:1:5
3 errors
